ios/AddCalendarEvent.m

```diff
@@ -152,6 +153,9 @@ -(EKEvent*)createNewEventInstance {
     if (options[_notes]) {
         event.notes = [RCTConvert NSString:options[_notes]];
     }
+    if (options[_allDay] && options[_allDay] != [NSNull null]) {
```

Comment:
Hello,
If for some reason allDay il null `options[_allDay] != [NSNull null]` prevent app crash
"-[NSNull boolValue]: unrecognized selector"
"JSON value '<null>' of type NSNull cannot be converted to BOOL"

According Apple documentation it can be used in situations where nil is prohibited as a value
https://developer.apple.com/library/content/documentation/Cocoa/Conceptual/NumbersandValues/Articles/Null.html